.github/workflows/tests.yml

```diff
@@ -8,15 +8,16 @@ jobs:
   test-php:
     name: Test on PHP ${{ matrix.php-version }} and ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    continue-on-error: ${{ matrix.experimental }}
+    continue-on-error: ${{ matrix.php-version == '8.5' }}
```

Comment:
I am not very fan of this change, it was practical to be able to have experimental versions